tests/vcs/git/GitCommitBearTest.py

```diff
@@ -100,6 +100,33 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_github_merge_commit(self):
+        _, __ = run_shell_command('echo "a" >> testfile1.txt ')
```

Comment:
I think we can just run it without saving it in `_`, `__`, the result isn't used anywhere